app/editor/src/features/content/papers/Papers.tsx

```diff
@@ -120,21 +117,24 @@ export const Papers: React.FC<IPapersProps> = (props) => {
   const fetch = React.useCallback(
     async (filter: IPaperFilter & Partial<IContentListAdvancedFilter>) => {
       try {
-        setLoading(true);
+        setIsLoading(true);
         const data = await findContent(
           makeFilter({
             ...filter,
           }),
         );
-        return new Page(data.page - 1, data.quantity, data?.items, data.total);
+        const page = new Page(data.page - 1, data.quantity, data?.items, data.total);
+        channel('page', page);
+        return page;
       } catch (error) {
         // TODO: Handle error
         throw error;
       } finally {
-        setLoading(false);
+        setIsLoading(false);
       }
     },
-    [findContent],
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
What is being ignored in the dependencies?  We always add documentation to ensure we're not introducing a hidden bug.